xblock/__init__.py

```diff
@@ -2,29 +2,4 @@
 XBlock Courseware Components
 """
 
-# For backwards compatibility, provide the XBlockMixin in xblock.fields
-# without causing a circular import
-import codecs
-import os
-import warnings
-
-import xblock.core
-import xblock.fields
-
-
-class XBlockMixin(xblock.core.XBlockMixin):
-    """
-    A wrapper around xblock.core.XBlockMixin that provides backwards compatibility for the old location.
-
-    Deprecated.
-    """
-    def __init__(self, *args, **kwargs):
-        warnings.warn("Please use xblock.core.XBlockMixin", DeprecationWarning, stacklevel=2)
-        super().__init__(*args, **kwargs)
-
-
-# For backwards compatibility, provide the XBlockMixin in xblock.fields
-# without causing a circular import
-xblock.fields.XBlockMixin = XBlockMixin
```

Comment:
We're just removing the wrapper locations, which have been deprecated since 2014. XBlockMixin is still available at xblock.core.XBlockMixin.